manifests/m/Microsoft/Sysinternals/TCPView/4.19/Microsoft.Sysinternals.TCPView.installer.yaml

```
@@ -1,16 +1,20 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: Microsoft.Sysinternals.TCPView
-PackageVersion: 4.18
+PackageVersion: 4.19
 Protocols:
 - tcp
 - udp
 - tcpv6
 - udpv6

```

```suggestion
- udpv6
Commands:
- tcpview
```
So that users can invoke the program by just typing tcpview in the terminal